provider/etcd/etcd.go

```diff
@@ -0,0 +1,196 @@
+// Copyright 2016 Prometheus Team
```

Comment:
SHOULD: Update this to say 2019.

Reply:
It's in all the headers like that.  I'll leave it.